src/posts/how-to-make-an-accessible-nav-with-dropdowns.md

```diff
@@ -62,26 +44,30 @@ Here's the example markup that we'll build up throughout this article. It's wort
 
 ### Semantics
 
-The &lt;nav&gt; element will communicate a role of navigation-equal to setting `role="navigation"`, (this may still need to be added if needing to [support some assistive technology](https://a11ysupport.io/tech/aria/navigation_role)) meaning it can be easily navigated to by people with screen readers—as a landmark.
+The <code>&lt;nav&gt;</code> element will communicate a role of navigation-equal to setting `role="navigation"`, (this may still need to be added if needing to [support some assistive technology](https://a11ysupport.io/tech/aria/navigation_role)) meaning it can be easily navigated to by people with screen readers—as a landmark.
```

Comment:
you don't need the parenthetical here.
1. support for nav being exposed as a landmark is very solid. 
2. the a11ysupport page you link to is for role=navigation and AT support for that role, so linking to it isn't even relevant here, since this page isn't for support of the element)